### PR TITLE
chore: prettify names

### PR DESCRIPTION
**Pull request by ecdsafu**
# Description of changes

Adds pretty names to wrappers, dexes and bridges

- closes ticket #

# Check specifically for:

WRITE_HERE

### CODE REVIEW

File: src/dexes/index.ts [hunk 1]
```diff
@@ -6,82 +6,129 @@ export interface Dex {
...
-  ZYBERSWAP = "Zyberswap",
-  ZYBERSWAP_V3 = "Zyberswap-v3",
+  ZYBERSWAP = "ZyberSwap V2",
+  ZYBERSWAP_V3 = "ZyberSwap V3",
 }
 
 // identifiers dex adapters
 export enum DexType {
   // EVM
-  ALGEBRA = "algebra",
```
Comment (jmdev3) [inline src/dexes/index.ts:71]: are we sure about this change?

File: src/dexes/index.ts [hunk 2]
```diff
@@ -6,82 +6,129 @@ export interface Dex {
...
-  PLATYPUS = "platypus",
-  PANCAKESWAP_STABLE = "pancakeswap-stable",
-  SADDLE = "saddle",
-  SOLIDLY = "solidly",
-  TRIDENT = "trident",
-  WOMBAT = "wombat",
-  UNISWAP_V2 = "uniswap-v2",
-  UNISWAP_V3 = "uniswap-v3",
-  MENTO_V2 = "mento-v2",
+
```
Comment (jmdev3) [inline src/dexes/index.ts:130]: are we sure about this change?

File: src/bridges/index.ts [hunk 3]
```diff
@@ -14,4 +14,20 @@ export enum BridgeProvider {
...
 
+export enum BridgeName {
+  AXELAR = "Axelar",
+  CCTP = "CCTP",
+  NOBLE_CCTP = "CCTP (Noble)",
+  IBC = "IBC",
+  PFM = "IBC forward",
+}
+
+export enum BridgeIcon {
```
Comment (jmdev3) [inline src/bridges/index.ts:25]: we should add this on the config, not here as it's a type package

File: src/dexes/index.ts [hunk 4]
```diff
@@ -6,82 +6,129 @@ export interface Dex {
 }
 
 // identifiers for dexes
+export enum DexIcon {
```
Comment (jmdev3) [inline src/dexes/index.ts:9]: same as above

**Review by jmdev3 — CHANGES_REQUESTED** [review]
leave some comments

File: src/dexes/index.ts (hunk 2 above)
Comment (ecdsafu) [reply src/dexes/index.ts:130]: No this is a mistake. DexType should not be deleted.

**Pull request closed without merging**